engines/supernova/screen.cpp

```diff
@@ -535,6 +532,10 @@ void Screen::renderMessage(const char *text, MessagePosition position, int posit
 	int y = 0;
 	byte textColor = 0;
 
+	_ttsMan = g_system->getTextToSpeechManager();
+	if (_ttsMan != nullptr && ConfMan.getBool("tts_enabled"))
+		_ttsMan->say(text,  Common::TextToSpeechManager::QUEUE_NO_REPEAT, Common::kDos850);
+
```

Comment:
Either set `_ttsMan` in the `Screen` constructor, or change it to a local variable.
But again I don't see the benefit of having this as a class member variable, so I would suggest making it a local variable.